editor/debugger/editor_visual_profiler.cpp

```diff
@@ -423,9 +425,18 @@ void EditorVisualProfiler::_clear_pressed() {
 	_update_plot();
 }
 
+void EditorVisualProfiler::_autostart_pressed() {
+	autostart_button->release_focus();
+	ProjectSettings::get_singleton()->set("debug/settings/profiler/autostart_visual_profiler", autostart_button->is_pressed());
```

Comment:
Use dedicated method instead of magic setter:

```suggestion
	ProjectSettings::get_singleton()->set_setting("debug/settings/profiler/autostart_visual_profiler", autostart_button->is_pressed());
```